README

```diff
@@ -51,9 +56,9 @@ the following in your local.conf
   ACE_TAO_OPTION = ""             to build with OCI ACE/TAO 2.2a with the latest patches
   ACE_TAO_OPTION = "--doc-group"  to build with DOC Group ACE 6.5.8 / TAO 2.5.8
 
-If PREFERRED_VERSION is not specifed in your local.conf then the OpenDDS 3.22 recipe will be
+If PREFERRED_VERSION is not specifed in your local.conf then the OpenDDS 3.24 recipe will be
 selected for build by default due to it having a higher version number and when ACE_TAO_OPTION
-is not defined the OCI ACE/TAO 2.2a with the latest patches option will be selected by default.
+is not defined the doc-group option will be selected by default.
```

Comment:
This describes the behavior of the 3.24 configure script but not the older configure scripts.  I haven't looked at the details of the scripts but I'd assume that the opendds_3.20.bb (or similar) match that behavior.

Reply:
Right, the opendds_3.xx script uses the configure script options to download ACE/TAO, so the opendds_3.xx script pre downloads the the ACE/TAO version that the configure script would have downloaded. I can remove script support for just set it up to download Docgroup ACE/TAO 6.5.19 and forego script support for Docgroup ACE/TAO 7.1.0